app/src/components/CustomEmoji.tsx

```diff
@@ -0,0 +1,99 @@
+import { styled } from "linaria/react";
+import { transparentize } from "polished";
+import React from "react";
+
+import { OtherColors } from "@app/theme/color";
+import { snowflakeToDate } from "@app/utility/discord";
+import { CustomEmoji, Member, Snowflake } from "@app/utility/types";
+import Tooltip from "@architus/facade/components/Tooltip";
+import {
+  isDefined,
+  formatDateExtraShort,
+  formatNum,
+} from "@architus/lib/utility";
+
+const Styled = {
+  TooltipName: styled.strong`
+    display: block;
+  `,
+  TooltipElevated: styled.div`
+    opacity: 0.7;
+    font-size: 80%;
+  `,
+  OuterContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+  `,
+  ImageContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100%;
+  `,
+
+  Image: styled.img`
+    max-width: 100%;
+    max-height: 100%;
+    object-fit: contain;
+  `,
+  Mention: styled.div`
+    max-width: max-content;
+    color: ${OtherColors.Discord};
+    background-color: ${transparentize(0.85, OtherColors.Discord)};
+
+    &:hover {
+      color: white;
+      background-color: ${transparentize(0.25, OtherColors.Discord)};
+    }
+  `,
+};
+
+type CustomEmojiProps = {
+  emoji: CustomEmoji;
+  author?: Member;
+  className?: string;
+  style?: React.CSSProperties;
+};
```

Comment:
Can you add a newline here :) and a doc comment explaining what the component does